<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Welcom to my personal page!</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo"><strong>Welcome to Ji-Ping Wang's webpage!</strong> </a>
									<ul class="icons">
													</ul>
								</header>

							<!-- Banner -->
								<section id="banner">
                                  <span class="image object">
										<img src="images/jiping_big_head5.jpg" alt=""  />
									</span>
									<div class="content">
										<header>
											<h1>Ji-Ping Wang</h1>
											<p>email: jzwang at northwestern dot edu </p>
										</header>
										<p>I am a Professor of Statistics and Data Science, an adjunct Professor of Molecular BioSciences. I am also a faculty member of the NSF-Simons Institute of Quantitative Biology at Northwestern University. I received my PhD in Statistics from Penn State in 2003.</p>
                                        
                                        <p>In my spare time, I like fishing, playing table tennis and hiking. </p>
										<ul class="actions">
											<li><a href="https://statistics.northwestern.edu/people/faculty/ji-ping-wang.html" class="button big">Learn More</a></li>
										</ul>
									   </div>
                                 

                                        
								</section>

							<!-- Section -->
								<section>
									<header class="major">
										<h2>My Research Interest</h2>
									</header>
                                    <p> Bioinformatics and computational biology; 
                                                Nucleosome mapping and positioning prediction; Epigenetics;
                                                Species number and populuation size estimation;
                                                    Mixture models and computing algorithms.</p>
                                       
									<div class="features">
                                        	<article>
										<!--		<span class="icon solid fa-signal"></span>-->
								 		<div class="content">
											 <ul class="actions">
											<li><a href="https://scholar.google.com/citations?hl=en&user=pijDrQMAAAAJ&view_op=list_works" class="button big">My publications</a></li>
										</ul> 
                                        </div>  
										</article>		
                                    
									</div>
								</section>

							<!-- Section -->
								<section>
									<header class="major">
										<h2>Some Bioinformatic Tools We Developped</h2>
									</header>
									<div class="posts">
                                       	<article>
											<a href="#" class="image"><img src="./images/crispr.jpg" alt="" /></a>
											<h2>BoostMEC</h2>
                                            <p><h3>BoostMEC is a boosting tree based approach, utilizing LightGBM for the prediction of wild-type CRISPR-Cas9 editing efficiency.</h3> CRISPR-Cas9 is the cutting-edge genome editting techonology. A variety of sgRNA properties have been found to be predictive of CRISPR cleavage efficiency, including the position-specific sequence composition of sgRNAs, global sgRNA sequence properties, and thermodynamic features. BoostMEC maintains an advantage over other state-of-the-art CRISPR efficiency prediction models that are based on deep learning through its ability to produce more interpretable feature insights and predictions. </p>
											<ul class="actions">
												<li><a href="https://github.com/oazarate/BoostMEC" class="button">Github</a></li>
											</ul>
										</article>
                                        <article>
											<a href="./images/ribo.jpg" class="image"><img src="./images/ribo.jpg" alt="" /></a>
											<h2>RiboDiPA</h2>
                                            <p><h3>RiboDiPA is a bioinformatics pipeline developed for differential pattern analysis of Ribo-seq footprint data.</h3> Ribosome is a protein complex that binds to mRNA sequence to facilitate translation. Its footprint, collected by ribo-seq assay, provides a snapshot of translation activities in cells in vivo. </p>
											<ul class="actions">
	                                           <li><a href="https://www.bioconductor.org/packages/release/bioc/html/RiboDiPA.html" class="button">Bioconductor</a></li>
                                                <li><a href="https://github.com/jipingw/RiboDiPA" class="button">Github</a></li>
																				</ul>
										</article>
										
                                        <article>
											<a href="#" class="image"><img src="./images/bend2.jpg" alt="" /></a>
											<h2>DNAcycP</h2>
                                            <p><h3>DNAcycP is a Python package for DNA cyclizability or bendability prediction trained based on loop-seq data.</h3> DNA mechanical properties play a critical role in every aspect of DNA-dependent biological processes. Recently a high throughput assay named loop-seq has been developed to quantify the intrinsic bendability of a massive number of DNA fragments simultaneously. DNAcycP, built on a deep-learning approach, predicts intrinsic DNA cyclizability with high fidelity compared to the experimental data. </p>
											<ul class="actions">
												<li><a href="https://github.com/jipingw/DNAcycP" class="button">Github</a></li>
                                                <li><a href="https://cyclizability.stats.northwestern.edu/" class="button">Web server</a></li>
											</ul>
										</article>
								       <article>
											<a href="./images/degnorm.jpg" class="image"><img src="./images/degnorm.jpg" alt="" /></a>
                                            <h2>DegNorm</h2>
											<p><h3>DegNorm is a bioinformatics pipeline to correct for bias due to the heterogeneous patterns of transcript degradation in RNA-seq data.</h3> RNA degradation is a commonly used measure for sample quality in RNA-seq, which is sample and gene-specific. DegNorm provides a sample- and gene-specific correction of bias due to degradation. </p>
											<ul class="actions">
												<li><a href="https://www.bioconductor.org/packages/release/bioc/html/DegNorm.html" class="button">Bioconductor</a></li>
                                                <li><a href="https://github.com/jipingw/DegNorm" class="button">Github</a></li>
											</ul>
										</article>
										<article>
											<a href="./images/butterfly.jpg" class="image"><img src="https://images.pexels.com/photos/1190128/pexels-photo-1190128.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2" alt="" /></a>
											<h2>SPECIES</h2>
											<p><h3>SPECIES is an R package that implements various popular methods in species richness estimation.</h3> How many butterfly species are there in a ecosystem? What was the vocabulary size of Shakespere? How many microbial species are there in human's gut system? All of these questions are related to a problem known as species number estimation problem.</p>
											<ul class="actions">
												<li><a href="https://cran.r-project.org/web/packages/SPECIES/index.html" class="button">CRAN</a></li>
                                                <li><a href="https://github.com/jipingw/SPECIES" class="button">Github</a></li>
                                                
											</ul>
										</article>
										<article>
											<a href="./images/nucleosome3.jpg" class="image"><img src="./images/nucleosome3.jpg" alt="" /></a>
											<h2>NuPoP and NuPoP_Fortran</h2>
											<p><h3>NuPoP is an R packageand NuPoP_Fortran is a Fortran program for Nucleosome Positioning Prediction and visualization based on DNA sequence.</h3> Nucleosome is the foundamental packing unit of DNA in eukaryotic cells.
                                            Its positioing is largely determined by intrinsic code in the DNA sequence. Nucleosome positioning regulates varous of cell activities such as transcription by controlling accessibity of protein factor binding.</p>
											<ul class="actions">
												<li><a href="https://github.com/jipingw/NuPoP" class="button">Bioconductor </a></li>
                                                <li><a href="https://github.com/jipingw/NuPoP" class="button">Github</a></li>
                                                </ul>
                                              <ul class="actions">  
                                                <li><a href="https://github.com/jipingw/NuPoP_doc" class="button">NuPoP doc</a></li>
                                                <li><a href="https://github.com/jipingw/NuPoP_Fortran" class="button">NuPoP_Fortran</a></li>
											</ul>
										</article>

										
									</div>
								</section>

						</div>
					</div>
            
                
				
		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>