<!DOCTYPE HTML>
<!--
	Editorial by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Ji-Ping Wang's Webpage</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">
						<div class="inner">

							<!-- Header -->
								<header id="header">
									<a href="index.html" class="logo"><h2>Ji-Ping Wang</h2> </a>
									<ul class="icons">
													</ul>
								</header>
                       
							<!-- Banner -->
								<section id="banner">
                                    
									<div class="content">
								
                                        
										<p>
                                            <span class="image right"><img src="images/jiping_big_head5.jpg" alt="" /></span>
                                            I am a Professor of <a href="https://statistics.northwestern.edu/">Statistics and Data Science</a>, an adjunct Professor of <a href="https://molbiosci.northwestern.edu/">Molecular BioSciences</a> and a faculty member of the <a href="https://www.nitmb.org/"> National Institute for Theory and Mathematics in Biology (NITMB)  </a> at Northwestern University. I received my PhD in Statistics from Pennsylvania State University in 2003 and BS from University of Science and Technology of China in 1995.
                                        </p>
                                        
                                        <p><b style="font-size:15px;">My research interest</b>: Bioinformatics and computational biology; 
                                                Nucleosome mapping and positioning prediction; Epigenetics;
                                                Species number and populuation size estimation;
                                                    Mixture models and computing algorithms</p>
                                        
                                        <p>In my spare time, I like fishing, playing pickleball, table tennis and hiking. </p>
										<ul class="actions">
											<li><a href="https://statistics.northwestern.edu/people/faculty/ji-ping-wang.html" class="button big">Learn More</a></li>
										</ul>
                                    </div>
								</section>
                            
                            <header class="major">
										<h2>News</h2>
									</header>
                            <div class="box">
                            <li><strong>BoostMEC</Strong>: predicting CRISPR-Cas9 cleavage efficiency through boosting models, 2022, <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04998-z" target="_top"> <em>NAR</em>,</a>  download <a href="https://github.com/oazarate/BoostMEC" target="_top"> Python package </a></li> 
                            <li><strong>DNAcycP</strong>: a deep learning tool for DNA cyclizability prediction, 2022, <a href="https://doi.org/10.1093/nar/gkac162" target="_top"><em>NAR</em>,</a> Python package <a href="https://github.com/jipingw/DNAcycP" target="_top">GitHub.</a> Real-time cyclizabilty prediction <a href="https://cyclizability.stats.northwestern.edu/" target="_top"> web server</a>.</li>
                            <li><strong>NuPoP 2.0 major update</strong>: nucleosome positioning prediction function using chemical map added, see <a href="https://github.com/jipingw/NuPoP" target="_top">GitHub,</a> download package from <a href="https://bioconductor.org/packages/release/bioc/html/NuPoP.html" target="_top"> Bioconductor </a>.</li>
                            </div>

                           		<!-- Section -->
								<section>
									<header class="major">
										<h2>Some Bioinformatic Tools We Developped</h2>
									</header>
									<div class="posts">
                                            <article>
											<h2>NuPoP <sup style="color:red;">update</sup></h2>
                                            <a href="./images/nucleosome3.jpg" class="image"><img src="./images/nucleosome3.jpg" alt="" /></a>
                                                <h3><p>NuPoP is an R package for Nucleosome Positioning Prediction based on DNA sequence. The model is trained based on in vivo chemical map or Mnase map of nucelosomes. </p></h3> Nucleosome is the foundamental packing unit of DNA in eukaryotic cells.
                                            Its positioing is largely determined by intrinsic code in the DNA sequence. Nucleosome positioning regulates varous of cell activities such as transcription by controlling accessibity of protein factor binding. See <span style="color:red;font-size:15px;">update</span> at the github site below.
											<ul class="actions">
												<li><a href="https://bioconductor.org/packages/release/bioc/html/NuPoP.html" class="button">Bioconductor </a></li>
                                                <li><a href="https://github.com/jipingw/NuPoP" class="button">Github</a></li>
                                                </ul>
                                              <ul class="actions">  
                                                <li><a href="https://github.com/jipingw/NuPoP_doc" class="button">NuPoP doc</a></li>
                                                <li><a href="https://github.com/jipingw/NuPoP_Fortran" class="button">NuPoP_Fortran</a></li>
											</ul>
										</article>
                                        <article>
                                            <h2>BoostMEC <sup style="color:red;">new</sup></h2>
                                            <a href="#" class="image"><img src="./images/crispr.jpg" alt="" /></a>
                                            <h3><p>BoostMEC is a boosting tree based approach, utilizing LightGBM for the prediction of wild-type CRISPR-Cas9 editing efficiency.</p></h3><p>CRISPR-Cas9 is the cutting-edge genome editting techonology. A variety of sgRNA properties have been found to be predictive of CRISPR cleavage efficiency, including the position-specific sequence composition of sgRNAs, global sgRNA sequence properties, and thermodynamic features. BoostMEC maintains an advantage over other state-of-the-art CRISPR efficiency prediction models that are based on deep learning through its ability to produce more interpretable feature insights and predictions.</p>
											<ul class="actions">
												<li><a href="https://github.com/oazarate/BoostMEC" class="button">Github</a></li>
											</ul>
										</article>
                                          <article>
                                            <h2>DNAcycP <sup style="color:red;font-size:20px;">new</sup> </h2>
											<a href="#" class="image"><img src="./images/bend2.jpg" alt="" /></a>
                                            <h3><p>DNAcycP has a Python package and a web server for DNA cyclizability or bendability prediction trained based on loop-seq data.</p></h3> DNA mechanical properties play a critical role in every aspect of DNA-dependent biological processes. Recently a high throughput assay named loop-seq has been developed to quantify the intrinsic bendability of a massive number of DNA fragments simultaneously. DNAcycP, built on a deep-learning approach, predicts intrinsic DNA cyclizability with high fidelity compared to the experimental data. 
											<ul class="actions">
												<li><a href="https://github.com/jipingw/DNAcycP" class="button">Github</a></li>
                                                <li><a href="https://cyclizability.stats.northwestern.edu/" class="button">Web server</a></li>
											</ul>
										</article>
										<article>
                                            <h2>SPECIES</h2>
											<a href="./images/butterfly.jpg" class="image"><img src="https://images.pexels.com/photos/1190128/pexels-photo-1190128.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2" alt="" /></a>
											<p><h3>SPECIES is an R package that implements various popular methods in species richness estimation.</h3> How many butterfly species are there in a ecosystem? What was the vocabulary size of Shakespere? How many microbial species are there in human's gut system? All of these questions are related to a problem known as species number estimation problem.
											<ul class="actions">
												<li><a href="https://cran.r-project.org/web/packages/SPECIES/index.html" class="button">CRAN</a></li>
                                                <li><a href="https://github.com/jipingw/SPECIES" class="button">Github</a></li>            
											</ul>
										</article>	
                                        
                                        
                                        <article>
                                            <h2>RiboDiPA</h2>
											<a href="./images/ribo.jpg" class="image"><img src="./images/ribo.jpg" alt="" /></a>
                                            <h3><p>RiboDiPA is a bioinformatics pipeline developed for differential pattern analysis of Ribo-seq footprint data.</p></h3> Ribosome is a protein complex that binds to mRNA sequence to facilitate translation. Its footprint, collected by ribo-seq assay, provides a snapshot of translation activities in cells in vivo. 
											<ul class="actions">
	                                           <li><a href="https://www.bioconductor.org/packages/release/bioc/html/RiboDiPA.html" class="button">Bioconductor</a></li>
                                                <li><a href="https://github.com/jipingw/RiboDiPA" class="button">Github</a></li>
																				</ul>
										</article>
									
								       <article>
                                            <h2>DegNorm</h2>
											<a href="./images/degnorm.jpg" class="image"><img src="./images/degnorm.jpg" alt="" /></a>
                                           <h3><p>DegNorm is a bioinformatics pipeline to correct for bias due to the heterogeneous patterns of transcript degradation in RNA-seq data.</p></h3> RNA degradation is a commonly used measure for sample quality in RNA-seq, which is sample and gene-specific. DegNorm provides a sample- and gene-specific correction of bias due to degradation. 
											<ul class="actions">
												<li><a href="https://www.bioconductor.org/packages/release/bioc/html/DegNorm.html" class="button">Bioconductor</a></li>
                                                <li><a href="https://github.com/jipingw/DegNorm" class="button">Github</a></li>
											</ul>
										</article>
                                        						
									</div>
								</section>

						</div>
					</div>

				<!-- Sidebar -->
					<div id="sidebar">
						<div class="inner">

							<!-- Menu -->
								<nav id="menu">
									<header class="major">
										<h2>Menu</h2>
									</header>
									<ul>
										<li><a href="index.html">Homepage</a></li>
										<li><a href="publication.html",target="_self">Publications</a></li>
										<li><a href="https://scholar.google.com/citations?hl=en&user=pijDrQMAAAAJ&view_op=list_works">Google Scholar</a></li>
                                        <li><a href="http://github.com/jipingw">My Github page</a></li>			
										
									</ul>
                                    </nav>
                               <!-- Section -->
								<section>
									<header class="major">
										<h2>Contact</h2>
									</header>
									<p>Address: Room 302, 2006 Sheridan Road, Evanston, IL 60208</p>
									<ul class="contact">
										<li class="icon solid fa-envelope"><a href="#">jzwang@northwestern.edu</a></li>
										<li class="icon solid fa-phone">(847) 467-6896</li>
									
									</ul>
								</section>


						</div>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
