<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
  <meta http-equiv="Content-Type"
 content="text/html; charset=iso-8859-1">
  <title>Ji-Ping Wang's pub</title>
  <style type="text/css">
<!--
body {
	<!background-image:  url(gray-wev.jpg);
}
body,td,th {
	font-size: 16px;
}
.style16 {font-family: Verdana, Arial, Helvetica, sans-serif}
.style7 {
	color: #0000FF;
	font-family: Verdana, Arial, Helvetica, sans-serif;
	font-size: 12px;
}
-->
  </style>
  <script language="JavaScript" type="text/JavaScript">
<!--



function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
//-->
</script>
</head>
<body>
<p class="style7">&nbsp;</p>
<p class="style7">&nbsp;</p>
<p class="style7">&nbsp;</p>
<p class="style7">Selected peer-reviewed publications</p>
<ul style="margin-top: 0; margin-bottom: 0; font-family: Geneva, Arial, Helvetica, sans-serif; font-size: 12px;">

<li class="style16">
<p>BoostMEC: predicting CRISPR-Cas9 cleavage efficiency through boosting models <br>
Zarate, O.A., Yang, Y., Wang, X. <strong> Wang, J.-P.</strong> <em><strong> BMC Bioinformatics</strong></em>,
23, 446 (2022). https://doi.org/10.1186/s12859-022-04998-z<br>
Python package <a href="https://github.com/oazarate/BoostMEC" target="_top"> download </a>
</li>

<li class="style16">
<p>DNAcycP: a deep learning tool for DNA cyclizability prediction <br>
Li, K., Caroll,M.,Vafabakhsh, R., Wang, X., <strong> Wang, J.-P.</strong> <em><strong> Nucleic Acids Research</strong></em>, 2022, https://doi.org/10.1093/nar/gkac162<br> Python package download: <a href="https://github.com/jipingw/DNAcycP" target="_top">GitHub,</a> Run DNA bendabilty/cyclizabilty prediction in real time <a href="https://cyclizability.stats.northwestern.edu/" target="_top"> web server,</a>
</li>

<li class="style16">
<p> G-DAug: Generative Data Augmentation for Commonsense Reasoning, <br>
Yang, Y., Malaviya, C., Fernandez, J., Swayamdipta, W., Le Bras, R., <strong> Wang, J.-P. </strong>, Bhagavatula, C., Choi, Y. and Downey, D., 
<em><strong>Proceedings of the 2020  Conference on Empirical Methods in Natural Language Processing</strong></em></strong>, 2020,  p1008-1025<br>
</li>


<li class="style16">
<p>RiboDiPA: A novel  tool for differential pattern analysis in Ribo-seq data <br>
Li, K., Hope, M., Wang, X., <strong> Wang, J.-P.</strong> <em><strong> Nucleic Acids Research</strong></em>, 2020,48(21), doi:10.1093/nar/gkaa1049<br>
</li>


<li class="style16">
<p>Mobilizable antibiotic resistance genes are present in dust microbial communities <br>
Maamar, S.B., Glawe, A.J., Brown, T.K., Hellgeth, N., Hu, J., <strong> Wang, J.-P.</strong>,Huttenhower, C. and Hartmann, E.M., 
<em><strong> PLoS Pathog.</strong></em>, 2020, 16(1): e1008211. https://doi.org/10.1371/journal.ppat.1008211.<br>
</li>

<li class="style16">
<p>Using Large Corpus N-gram Statistics to Improve Recurrent Neural Language Models,<br>
Yang,Y., <strong> Wang, J.-P.</strong>, Downey, D.,
<em><strong>
Proceedings of the 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies, (Volume 1: Long and Short Papers),
</strong></em>, 2019, NAACL,3268-3273, Minneapolis, Minnesota.
</li>


<li class="style16">
<p>DegNorm: normalization of generalized transcript degradation improves accuracy in RNA-seq analysis<br>
Xiong, B., Yang, Y., Fineis, F.<strong> Wang, J.-P.</strong>,
<em><strong> Genome Biology</strong></em>, 2019, 20:75.<br>
DegNorm python package at  <span class="style16"><a href="https://nustatbioinfo.github.io/DegNorm/"><em><strong> Github</em></strong> </a><span>
</li>



<li class="style16">
<p> An Integrative Classification Model for Multiple Sclerosis Lesion Detection in Multimodal MRI<br>
Zhang, F., <strong> Wang, J.-P.</strong>, Jiang, W.
<em><strong> Statistics and Its Interface</strong></em>, 2019, 12(2).
</li>


<li class="style16">
<p> Extracting Commonsense Properties from Embeddings with Limited Human Guidance <br>
Yang,Y., Birnbaum, L., <strong> Wang, J.-P.</strong>, Downey, D.
<em><strong>. Proceedings of the 56th Annual Meeting of the Association for Computational Linguistics (Volume 2: Short Papers) </strong></em>,2018, Association for Computational Linguistics, 644--649, Melbourne, Australia
</li>



<li class="style16">
<p> Genome-wide mapping of the nucleosome landscape by micrococcal nuclease and chemical mapping <br>
Voong,L. N., Xi, L.,  <strong> Wang, J.-P.</strong>, Wang, X.
<em><strong>. Trends in Genetics</strong></em>, 2017,33(8),495-507  DOI: http://dx.doi.org/10.1016/.tig.2017.05.007.
</li>

<li class="style16">
<p> Insights into Nucleosome Organization in Mouse Embryonic Stem Cells through Chemical Mapping <br>
Voong,L. N., Xi, L., Sebeson, A.C., Xiong, B., <strong> Wang, J.-P.</strong>, Wang, X.
<em><strong>. Cell</strong></em>, 2016, 167(6),1555-1570.e15, DOI: http://dx.doi.org/10.1016/j.cell.2016.10.049.
highlighted in <span class="style16"><a href="http://www.nature.com/nrm/journal/v18/n1/full/nrm.2016.167.html"><em><strong> Nature Reviews Molecular Cell Biology</em></strong> </a><span> <br>
</li>

<li class="style16">
<p> A Bayesian probit model with spatially varying coefficients for brain decoding using fMRI data <br>
Zhang, F., Jiang, W.,Wong, P.W., <strong> Wang, J.-P.</strong>,
<em><strong> Statistics in Medicine</strong></em>, 2016,10.1002/sim.6999.
</li>

<li class="style16">
<p>Set1/COMPASS and Mediator are repurposed to promote epigenetic transcriptional memory, <br>
D'Urso, A ., Yoh-hei Takahashi, Y.-H., Xiong, B., Marone, J.,  Coukos, R., Randise-Hinchliff, C., <strong>Wang, J.-P.</strong>, Shilatifard, A., and Brickner, J.H
<em><strong> eLife</strong></em>, 2016,10.7554/eLife.16691.
</li>

<li class="style16">
<p>Involvement of PARP-1 in the regulation of alternative splicing <br>
Elena, M., Maiorano, J., Zhang, Q., Convertini, P., Stamm, S.,<strong> Wang, J.-P.</strong>, and Yvonne N. Fondufe-Mittendorf
<em><strong> Cell Discovery</strong></em>, 15046(2016).
</li>

<li class="style16">
<p>Differential Nucleosome Occupancies across Oct4-Sox2 Binding Sites in Murine Embryonic Stem Cells, <br>
Sebeson,A., Xi, L., Zhang, Q.,Sigmund, A., <strong>Wang, J.-P.</strong>, Widom, J. and Wang, X.
<em><strong> PLoS ONE</strong></em>, 2015,DOI: 10.1371/journal.pone.0127214.
</li>

<li class="style16">
<p>Integrative network analysis of TCGA data for ovarian cancer, <br>
Zhang, Q., Burdette, J.E., <strong>Wang, J.-P.</strong>,                                   
<em><strong> BMC Systems Biology</strong></em>, 2014, 8:1338.
</li>


<li class="style16">
<p>Decoding multiple sound categories in the human temporal cortex using
high resolution fMRI, <br>
Zhang, F., <strong>Wang, J.-P.</strong>, Kim, J., Parrish, T., Wong, P.C.M.
<em><strong> PLoS ONE</strong></em>, 2015, 10(2):e0117303.
</li>



<li class="style16">
<p>Histone Methyltransferase MMSET/NSD2 Alters EZH2 Binding and Reprograms the Myeloma Epigenome through Global and Focal Changes in H3K36 and H3K27 Methylation, <br>
Popovic, R., Martinez-Garcia, E., Giannopoulou, E., Zhang, Q., Ezponda, T., Shah, M.Y., Zheng, Y., Will, C.M., Small, E.C., Hua, Y., Bulic, M., Jiang, Y., Carrara, M., Calogero, R.A., Kath, W.L., Kelleher, N.L., <strong>Wang, J.-P.</strong>, Elemento, O. and Licht, J.D., <em><strong> PLoS Genetics</strong></em> 2014,10(9):e1004566,DOI: 10.1371/journal.pgen.1004566
.
</li>

<li class="style16">
<p>Single-cell nucleosome mapping reveals the molecular basis of gene expression heterogeneity, <br>
Small, E.C., Xi, L., <strong>Wang, J.-P.</strong>, Widom, J., and Licht, J.D.,
<em><strong> PNAS</strong></em> 2014,111(24):E2462-71,doi: 10.1073/pnas.1400517111.
</li>


<li class="style16">
<p>The budding yeast Centromere DNA Element II wraps a stable Cse4 hemisome in either orientation in vivo, <br>
Henikoff, S., Ramachandran, S., Krassovsky, K., Bryson, E.D., Codomo, C.A., Brogaard, K., Widom, J., <strong>Wang, J.-P.</strong>, Henikoff, J.G.,
<em><strong> eLife</strong></em> 2014, 3:e01861.
</li>

<li class="style16">
<p>High Density Nucleosome Occupancy Map of Human Chromosome 9p21-22 Reveals Chromatin Organization of the Type I Interferon Gene Cluster, <br>
Freaney, J., Zhang, Q., Yigit, E., Kim, R., Widom, J., <strong>Wang, J.-P.</strong>, Horvath, C., <em><strong>Journal of Interferon & Cytokine Research</strong></em>, 2014,34(9):676-85
</li>

<li class="style16">
    <p>A locally convoluted cluster model for nucleosome positioning signals in chemical map<br>
Xi, L., Brogaard,K., Zhang, Q., Lindsay, B.G., Widom, and <strong>Wang, J.-P.</strong>, <em><strong>Journal of American Statistical Association </strong></em>, 
2014, 109(505) 48-62, DOI:10.1080/01621459.2013.862169
</li>

<li class="style16">
    <p>Chemical map of Schizosaccharomyces pombe reveals species-specific features in nucleosome positioning<br>
Moyle-Heyrman, G., Zaichuk, T., Xi, L., Zhang, Q., Uhlenbeck, O.C., Holmgren, R., Widom, J. and <strong>Wang, J.-P.</strong>,
<em><strong>PNAS</strong></em> 2013,110(50),20158-20163<a href="http://www.pnas.org/content/110/50/20158.full.pdf+html"> pdf</a>; <a href="SpombeCmap.html"> Supplementary materials </a>
  </li>

<li class="style16">
    <p>Quantifying copy number variations using a hidden Markov model with inhomogeneous emission distributions<br>
McCallum, K.J. and <strong>Wang, J.-P.</strong>, <em><strong>Biostatistics</strong></em>, 2013, Jul;14(3):600-11. doi: 10.1093/biostatistics/kxt003
  </li>

<li class="style16">
    <p>Functional Specialization of the Small Interfering RNA Pathway in Response to Virus Infection<br>
Marques, J.T., <strong>Wang, J.-P.</strong>, Wang, X., de Oliveira, K.P.V., Gao, C., Aguiar, E.R.G.R.
Jafari,N., Carthew, R.W., <em><strong>PLoS Pathog</strong></em> 2013, 9(8): e1003579. doi:10.1371/journal.ppat.1003579</p> 
  </li>

<li class="style16">
    <p>Archaeal nucleosome positioning in vivo and in vitro is directed by primary sequence motifs<br>
      Nalabothula,N., Xi,L., Bhattacharyya,S., Widom,J.,  <strong>Wang,  J.-P.</strong>, Reeve,NJ, Santangelo, JT, Fondufe-Mittendorf, NY, <em><strong>BMC Genomics</strong></em> 2013, 14:391 <a href="http://www.biomedcentral.com/1471-2164/14/391">PDF</a>
  </li>

 <li class="style16">
    <p>High-resolution nucleosome mapping of targeted regions using BAC-based enrichment <br>
      Yigit, E., Zhang, Q., Xi, L., Grilley, D., Widom, J., <strong>Wang, J.-P.</strong>, Rao, A. and Pipkin, M.E. <em><strong>Nucleic Acids Res.</strong></em> 2013;doi: 10.1093/nar/gkt081 <a href="http://nar.oxfordjournals.org/content/early/2013/02/14/nar.gkt081.full.pdf+html"> PDF</a>;<a href="Yigit2013NAR.html" target="_self">Supplementary materials</a>
  </li>

 <li class="style16">
    <p>Nucleosome mapping across the CFTR locus identifies novel regulatory factors <br>
      Yigit, E., Bischof, J.M., Zhang, Z., Ott, C.J., Kerschner, J.L., Leir, S.H., Buitrago-Delgado, E., Zhang, Q., <strong>Wang, J.P.</strong>, Widom, J., Harris, A. <em><strong>Nucleic Acids Res.</strong></em> 2013;doi: 10.1093/nar/gks1462 <a href="http://nar.oxfordjournals.org/content/early/2013/01/15/nar.gks1462.full.pdf+html"> PDF</a>
  </li>

  <li class="style16">
    <p>A chemical approach to mapping nucleosomes at base pair resolution <br>
      Brogaard, K., Xi, L., <strong>Wang, J.-P.</strong>, and Widom, J., <em><strong>Methods Enzymol.</strong></em> 2012;513:315-34</p>
  </li>
  <li class="style16">A Map of Nucleosome Positions in Yeast at base-pair resolution<br>
    Brogaard, K., Xi, L., <strong>Wang, J.-P.</strong>, and Widom, J., <strong><em>Nature</em></strong>, 2012, 486: 496&ndash;501.<br>
    <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature11142.html">Online Supplementary Methods</a>; <a href="NuCmap.html" target="_self">Other supplementary materials</a><br>
    <p></p>
  </li>
  <li class="style16">Large-scale Cortical Network Properties Predict Future Sound-to-Word Learning Success </li>
</ul>
<ol style="margin-top: 0; margin-bottom: 0; font-family: Verdana, Arial, Helvetica, sans-serif; font-size: 12px;">
  <p style="margin-top: 0;">Sheppard, J.P., <strong>Wang, J.-P.</strong> and Wong, P.C.M., <em><strong>Journal of Cognitive Neuroscience</strong></em>, 2012, 24(5), 1087-1103, <a href="http://www.mitpressjournals.org/doi/abs/10.1162/jocn_a_00210">PDF</a> </p>
</ol>
<ul style="margin-top: 0; margin-bottom: 0; font-family: Geneva, Arial, Helvetica, sans-serif; font-size: 12px;">
  <li class="style16">SPECIES: An R package for Species Richness Estimation <br>
      <strong>Wang, J.P.</strong>,<strong><em> Journal of Statistical Software</em></strong>, 2011, 40(9),1-15 <a href="http://www.jstatsoft.org/v40/i09">PDF</a> <br>
      <p></p>
  </li>
  <li class="style16"><span style="margin-top:0;margin-bottom:0;">An Empirical Bayes Based Approach for</span><span style="">Methylation Differentiation at the Single Nucleotide Resolution</span> <br>
      <span style="">McCallum, K., Jiang, W. and&nbsp;<strong>Wang, J.-P.</strong></span>, <span style=""><strong><em>International Journal of Mathematics and Computer Science</em></strong>, 2010, 5(2) 87-100 <a href="McCallumWang2010.pdf" target="_blank">PDF</a><br>
      </span>
    <p></p>
  </li>
  <li class="style16">Large-scale Cortical Functional Organization and Speech Perception across the Lifespan<br>
    Sheppard, J.P., <strong>Wang, J.-P.</strong>,Wong, P.C.M., <em><strong>PLoS ONE</strong></em>, 2010, 6(1):e16510.doi:10.1371/journal.pone.0016510 <a href="http://www.plosone.org/article/info%3Adoi/10.1371/journal.pone.0016510#abstract0">PDF</a><br>
    <p></p>
  </li>
  <li class="style16">Predicting nucleosome positioning using a duration Hidden Markov Model<br>
    Xi ,L., Fondufe-Mittendorf , Y.,Xia, L., Flatow,J., Widom <strong>,</strong>J. and <strong>Wang, J.-P., </strong><br>
    <em><strong>BMC Bioinformatics </strong></em>, 2010, doi:10.1186/1471-2105-11-346, <a href="http://www.biomedcentral.com/1471-2105/11/346">PDF</a>, <a href="http://nucleosome.stats.northwestern.edu/" target="_blank">NuPoP software </a>&nbsp; <a href="http://www.bioconductor.org/help/bioc-views/release/bioc/html/NuPoP.html">Bioconductor download</a><br>
    <p></p>
  </li>
  <li class="style16">A Poisson-compound Gamma model for species richness estimation <br>
      <strong>Wang, J.-P.</strong> , <em><strong>Biometrika</strong></em>, 2010, 97(3): 727-740, <a href="OnlineSupp.pdf">suppplemental materials </a>&nbsp; <a href="SPECIES.htm">SPECIES_V1.0 R package </a><br>
      <p></p>
  </li>
  <li class="style16">Estimation in Capture-Recapture models when covariates are subject to measurement errors and missing data <br>
    Xi, L, Watson, R., <strong>Wang, J.-P. </strong> and Yip, P.S.F,<strong><em>The Canadian Journal of Statistics</em></strong> 2009, 37:645&ndash;658 <a href="documents/XiCJS2009_000.pdf">PDF<br>
    </a>
    <p></p> 
  </li>
  <li class="style16">A flexible Statistical Model for Chromatin Sequences<br>
      <strong>Wang, J.-P.</strong> and Xi, L., In<em><strong> Frontiers of Biostatistics and Bioinformatics</strong></em>, Ma, S., Wang, Y. (eds). University of Science and Technology of China Press, 2009, Anhui, P.R. China<br>
      <p></p>
  </li>
  <li class="style16">Preferentially quantized linker DNA lengths in Saccharomyces cerevisiae <br>
      <strong>Wang, J.-P.</strong>, Fondufe-Mittendorf,Y., Xi, L., Tsai, G.,Segal, E. and Widom, J., <em><strong>PLoS Computational Biology</strong></em>, 2008, 4(9) e1000175 <a href="http://www.ploscompbiol.org/doi/pcbi.1000175">PDF</a>, <a href="WangWidomPLoS08.html">Supplementary and codes</a><br>
      <p></p>
  </li>
  <li class="style16">An exponential partial prior for improving NPMLestimation for mixtures <br>
      <span style="font-weight: bold;">Wang, J.-P.,&nbsp;</span> and Lindsay, B.G., <span style="font-weight:bold; font-style: italic;">Statistical Methodology</span>, 2008,5:30-45 <a href="http://dx.doi.org/10.1016/j.stamet.2007.03.004">PDF</a><br>
      <p></p>
  </li>
  <li class="style16">Meta-analysis of Drosophila Circadian Microarray Studies Identifies a Novel Set of Rhythmically Expressed Genes<br>
    Keegan, K., Pradhan, S. <strong>Wang, J.-P. </strong>and Allada, R., <span style="font-weight:bold; font-style: italic;">PLoS Computational Biology </span>, 2007, 3(11) e208, <a href="http://www.ploscompbiol.org/article/info:doi%2F10.1371%2Fjournal.pcbi.0030208"> PDF<br>
    </a>
    <p></p>
  </li>
  <li class="style16">Some optimization results and a VDM/ECM algorithm for penalized and constrained nonparametric maximum likelihood estimation for mixtures<br>
      <span style="font-weight: bold;">Wang, J.-P. , <em>Computational Statistics and Data Analysis</em></span> , 2007, 51:2946-2957 <a href="publishedpaper/WangCOMSTA2007.pdf">PDF</a><br>
      <p></p>
  </li>
  <li class="style16">A genomic code for nucleosome positioning<br>
    Segal, E., Fondufe-Mittendorf,Y., Chen,L., Th&aring;str&ouml;m, A., Field, Y., Moore, I.K., <span
 style="font-weight: bold;">Wang, J.-P.Z.</span> and Widom,J., <span style="font-weight: bold; font-style: italic;">Nature</span>, 2006, 442(7104):772-778<br>
    <p></p>
  </li>
  <li class="style16">Improved alignment of nucleosome DNA sequences using a mixture model<span style="font-weight: bold;"><br>
    Wang, J.-P. Z.</span> and Widom, J. , <em><strong>Nucleic Acids Research, </strong>2005 </em>, 33(21):6743-6755 2005, <a
 href="http://nar.oxfordjournals.org/cgi/reprint/33/21/6743?ijkey=xzgzMqdWFkUF5IS&amp;keytype=ref">PDF</a> <a
 href="http://bioinfo.stats.northwestern.edu/%7Ejzwang/publication/nucleosome/supp/suppNucleosome.html"> Supplementary and codes </a><br>
    <p></p>
  </li>
  <li class="style16">A penalized nonparametric maximum likelilhood approach to species richness estimation<span style="font-weight: bold;"><br>
    Wang, J.-P. Z.</span> and Lindsay, B. G., <span style="font-style: italic; font-weight: bold;">Journal of American Statistical Association</span>, 2005,100(471):942-959. <a href="publishedpaper/WangLindsay2005JASA.pdf">PDF </a> <br>
    <p></p>
  </li>
    
  <li class="style16">Gene capture prediction and overlap estimation in EST sequencing from one or multiple libraries<span
 style="font-weight: bold;"><br>
    Wang, J.-P. Z.</span>, Lindsay, B.G., Cui, L., Wall, P.K., Marion, J., Zhang, J. and W., dePamphilis, C., <strong><em>BMC Bioinformatics</em></strong>, 2005,6:300 <a
 href="http://www.biomedcentral.com/1471-2105/6/300/abstract">PDF</a> <a
 href="http://bioinfo.stats.northwestern.edu/%7Ejzwang/publication/GenePred/supp/suppGeneEst.html"> Supplementary and software </a> <br>
    <p></p>
  </li>
    
  <li class="style16">EST Clustering Error Evaluation and Correction <span
 style="font-weight: bold;"><br>
    Wang, J.-P. Z.</span>, Lindsay, B.G., Leebens-Mack, J. , Cui, L., Wall, P.K., Miller, W.C. and W., dePamphilis, C., <span style="font-weight: bold; font-style: italic;">Bioinformatics,</span> 2004, 20(17):2973-2984&nbsp; <a
 href="http://bioinformatics.oupjournals.org/cgi/content/abstract/bth342v1">PDF</a>&nbsp; <a
 href="http://bioinfo.stats.northwestern.edu/%7Ejzwang/publication/ESTError/supp/suppESTError.html">Supplementary and software </a>
    <p></p>
  </li>
</ul>
<ol style="margin-top: 0; font-family: Geneva, Arial, Helvetica, sans-serif; font-size: 12px;">
  <p style="margin-bottom: 0; font-family: Verdana, Arial, Helvetica, sans-serif;"></p>
</ol>
</body>
</html>
